### ISSUE TITLE
/etc/setup also gets run when we login as root

### ISSUE THREAD

**Issue by ket000** [WhitewaterFoundry/WLinux#87]
**Describe the bug**
/etc/setup also gets run when we login as root. It should only be run for non root user. Current workaround is to select N 2 times.

Also it does not work if your environment needs proxy setup. 

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Basic Troubleshooting Checklist**

[ ] I have confirmed WSL is enabled: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#confirm-wsl-is-enabled.
[ ] I have searched Google for the error message.
[ ] I have searched the official Microsoft WSL issues page: https://github.com/Microsoft/WSL/issues.
[ ] I have searched the WLinux issues page: https://github.com/WhitewaterFoundry/WLinux/issues.
[ ] I have reset WLinux: Settings->Apps->Apps & features->WLinux->Advanced Options->Reset. 
[ ] I have disabled and re-enabled WSL in Windows Features.
[ ] I have run Windows 10 updates and restarted.

What other troubleshooting have you attempted?

Insert here:

**WLinux Version**

Find: Settings->Apps->Apps & features->WLinux->Advanced Options->Version.

Insert here:

**Windows Build**

Run 'systeminfo | findstr /C:"OS"' in Command Prompt and insert here:

For help on retrieving: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#check-your-build-number

**Comment by sirredbeard** [comment]
When logged in as root, `$ touch .firstrun` to prevent setup from running again.

Sorry about that.

**Comment by ket000** [comment]
Thanks for the update. I figured out that. It seems that I got lot of issues with /etc/setup script when it tried to run automatically as soon as you try to login. I would recommend that you make it optional and do not run automatically and ask the user to run it manually. Also it may be advisable to make each additional step to be run individually rather than in one big script. I see several issues with the script and it does not seems to be ready for mainstream release. It also does not work in corporate environments due to proxy issue and need to setup proxy manually. 

Hopefully you will make changes in future versions to make it more robust. For not I am reading the script and running the steps manually.
